documentation/source/user_section/tutorials/spinal-nerve-rootlets-segmentation/rootlets-segmentation.rst

```diff
@@ -1,24 +1,25 @@
 Spinal nerve rootlets segmentation
 ##################################
 
-SCT provides a deep learning model for the segmentation of spinal nerve rootlets from T2*-weighted images.
-The model is available in SCT v6.2 and higher via ``sct_deepseg -task seg_spinal_rootlets_t2w``. In SCT v7.0, the command was changed to ``sct_deepseg rootlets_t2``.
+SCT provides a deep learning model for the segmentation of spinal nerve rootlets from T2-weighted and MP2RAGE images.
+The model is available in SCT v7.1 and higher via ``sct_deepseg rootlets``.
+In the previous SCT versions (SCT v6.2 and higher), the model segmented only T2-weighted and was available via via ``sct_deepseg -task seg_spinal_rootlets_t2w``.
```

Comment:
I like this rewording!

Though, do we want to wait for v7.1 to release this, or do we want to backport it to v7.0?

I think we should have time to backport it, especially given that we want to also backport an in-progress QC report fix (PR #4889).

Ideally, if we get that PR reviewed and merged today, I can have v7.0 ready by end of day?

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4895/commits/492049a92ca80ee7f16587766ff7ca9e16502059 :-)